maven-plugin/src/main/java/org/openmrs/maven/plugins/RemoveDependency.java

```diff
@@ -0,0 +1,61 @@
+package org.openmrs.maven.plugins;
+
+import org.apache.commons.lang.StringUtils;
+import org.apache.maven.plugin.MojoExecutionException;
+import org.apache.maven.plugin.MojoFailureException;
+import org.apache.maven.plugins.annotations.Mojo;
+import org.apache.maven.plugins.annotations.Parameter;
+import org.openmrs.maven.plugins.model.DistroProperties;
+
+import java.io.File;
+import java.util.Properties;
+import java.util.stream.Collectors;
+
+@Mojo(name = "rm-dependency", requiresProject = false)
+public class RemoveDependency extends AbstractTask {
+
+    /**
+     * Path to the openmrs-distro.properties file to modify
+     */
+    @Parameter(property = "distro")
+    private String distro;
+
+    /**
+     * Name of the property you want to remove
+     */
+    @Parameter(property = "property")
+    private String property;
+
+    @Override
+    public void executeTask() throws MojoExecutionException, MojoFailureException {
+
+        if (distro == null) {
+            File userDir = new File(System.getProperty("user.dir"));
+            File distroFile = new File(userDir, DistroProperties.DISTRO_FILE_NAME);
+            if (distroFile.exists()) {
+                distro = distroFile.getAbsolutePath();
+            }
+        }
```

Comment:
So what happens here if there's no distro file in the local directory and none supplied?